.github/workflows/test.yaml

```diff
@@ -71,8 +71,8 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           platforms: linux/amd64
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+          cache-from: type=registry,ref=ghcr.io/pluralsh/console:buildcache
```

Comment:
I think it's probably wise to use a slightly different repo name (pluralsh/console-cache) for this